recipes/dataframe/all/test_package/test_package.cpp

```diff
@@ -1,34 +1,82 @@
 #include <DataFrame/DataFrame.h>
+
 #include <cassert>
+#include <cmath>
+#include <iostream>
+#include <limits>
+#include <string>
+#include <typeinfo>
 
 using namespace hmdf;
 
 typedef StdDataFrame<unsigned long> MyDataFrame;
 
-int main(int argc, char *argv[]) {
+// -----------------------------------------------------------------------------
+
+static void test_haphazard()  {
+
+    MyDataFrame::set_thread_level(10);
+
+    std::cout << "\nTesting load_data ..." << std::endl;
+
+    MyDataFrame         df;
+    std::vector<int>    &col0 =
+        df.create_column<int>(static_cast<const char *>("col_name"));
+
+    std::vector<int>            intvec = { 1, 2, 3, 4, 5 };
+    std::vector<double>         dblvec =
+        { 1.2345, 2.2345, 3.2345, 4.2345, 5.2345 };
+    std::vector<double>         dblvec2 =
+        { 0.998, 0.3456, 0.056, 0.15678, 0.00345, 0.923, 0.06743, 0.1 };
+    std::vector<std::string>    strvec =
+        { "Col_name", "Col_name", "Col_name", "Col_name", "Col_name" };
+    std::vector<unsigned long>  ulgvec =
+        { 1UL, 2UL, 3UL, 4UL, 5UL, 8UL, 7UL, 6UL };
+    std::vector<unsigned long>  xulgvec = ulgvec;
+    const size_t                total_count =
+        ulgvec.size() +
+        intvec.size() +
+        dblvec.size() +
+        dblvec2.size() +
+        strvec.size() +
+        xulgvec.size() +
+        9;  // NaN inserterd
 
-  MyDataFrame::set_thread_level(10);
+    MyDataFrame::size_type  rc =
+        df.load_data(std::move(ulgvec),
+                     std::make_pair("int_col", intvec),
```

Comment:
both std::move and make_pair are part of utility, but is not listed

Reply:
https://github.com/conan-io/conan-center-index/pull/3195#issuecomment-713535531 upstream requires higher STL support

Reply:
Done